docs/tutorials/steps_to_train_models_with_multi_gpus_using_dp.md

```diff
@@ -0,0 +1,57 @@
+# Steps to train models with multiple gpus using DataParallel
+
+If you are interested in training deep forecasting models in parallel using multiple GPUs, please follow the instructions below.
+
+**Notes**: Currently, we only support **DataParallel**, and it is supported only on the **Sequential Backend**.
+
+
+### 1. Check the GPU devices information on your machine
+First and foremost, please ensure that there are multiple GPUs on your machine and that they are successfully detected. This can be accomplished in several ways:
+1) Using nvidia-smi command
+```
+nvidia-smi
+```
+2) Using torch.cuda.device_count()
+```
+import torch
+print("torch.cuda.device_count()")
```

Comment:
Here, `torch.cuda.device_count()` is a string.

Reply:
The document has been updated